image_classification/main.js

```diff
@@ -279,10 +279,10 @@ async function main() {
       console.log('- Computing... ');
       const computeTimeArray = [];
       let medianComputeTime;
-      if (numRuns > 1) {
-        // Do warm up
-        netInstance.compute(inputBuffer, outputBuffer);
-      }
+
```

Comment:
@Honry Regarding to WebNN-Polyfill, we actually warmup during compiling graph. For WebNN-Native, as we discussed if we do so, it would affect compile time, so current this modification is OK to add this warm up. @huningxin @fujunwei WDYT, thanks.